src/System.Reflection.DispatchProxy/tests/project.json

```diff
@@ -20,7 +22,6 @@
     "netstandard1.3": {}
   },
   "supports": {
-    "coreFx.Test.netcore50": {},
```

Comment:
Yes. It is required to test `raise_Xxx`because C# code cannot emit `raise_` methods of events.